packages/main-api/src/controllers/v3/notifications/list_notifications.rs

```diff
@@ -0,0 +1,36 @@
+use crate::{
+    features::notification::{ListNotificationsResponse, Notification, NotificationQueryOption, NotificationResponse},
+    types::{list_items_query::ListItemsQuery, Pagination},
+    *,
+};
+use aide::NoApi;
+use axum::{extract::{Query, State}, http::StatusCode, Json};
+use bdk::prelude::*;
+
+pub async fn list_notifications_handler(
+    State(AppState { dynamo, .. }): State<AppState>,
+    NoApi(user): NoApi<User>,
+    Query(Pagination { bookmark }): ListItemsQuery,
+) -> std::result::Result<Json<ListNotificationsResponse>, Error> {
+    let user_pk = &user.pk;
+
+    // Get all notifications for the user, ordered by created_at descending
+    let mut opt = NotificationQueryOption::builder()
+        .limit(100)
+        .scan_index_forward(false);
+
+    if let Some(bookmark) = bookmark {
+        opt = opt.bookmark(bookmark);
+    }
```

Comment:
```suggestion
    let opt = Notification::opt_with_bookmark(bookmark);
```

We are using 10 limits as default due to not enough space on a page.